hashcompute.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/corona10/goimagehash/etcs"
 	"github.com/corona10/goimagehash/transforms"
-	"github.com/nfnt/resize"
+	"github.com/disintegration/imaging"
```

Comment:
Would you like to check the possibility of using the old resizing library?
https://github.com/nfnt/resize/blob/83c6a9932646f83e3267f353373d47347b6036b2/resize.go#L48-L52